api/api.go

```diff
@@ -384,7 +386,8 @@ type CloudBackupStatusRequest struct {
 	// active on current node must be returned
 	Local bool
 	// Name of the backup/restore task. If this is specified, SrcVolumeID is
-	// ignored
+	// ignored. This could be GroupCloudBackupId too, and in that case multiple
+	// statuses belonging to the groupCloudBackupID is returned.
 	Name string
```

Comment:
Maybe have a GroupName instead of overloading this name?

Reply:
Not sure if adding the new field GroupName makes it simpler? client to specify either task Name or groupID in same field, removing the confusion. Adding new field would also mean status for combination of groupID-taskName.

Reply:
Can we at least rename this from `Name` to `ID` like it originally was then to make is clearer? (while maintaining backward compatibility) We pass in `Name` here but status is returning `ID`

Reply:
Yes. Changed it to ID while retaining the backward compatibility for clients using older api.